docker/conf/docker-compose-mac.yml.dist

```diff
@@ -21,9 +21,15 @@ services:
     webserver:
         image: nginx:1.13-alpine
         container_name: shopsys-framework-webserver
+        depends_on:
+            - adminer
+            - elasticsearch
+            - php-fpm
+            - redis-admin
+            - selenium-server
         volumes:
             - shopsys-framework-web-sync:/var/www/html/project-base/web
-            - ./docker/nginx/nginx.conf:/etc/nginx/conf.d/default.conf:delegated
+            - ./docker/nginx:/etc/nginx/conf.d:delegated
```

Comment:
this thing is only available for mac and its filesystem https://docs.docker.com/docker-for-mac/osxfs-caching/, win doesn't have it. Linux doesn't work in virtualized mode like mac or win so it would be useless for it.